src/NuGet.Core/NuGet.Frameworks/def/IFrameworkNameProvider.cs

```diff
@@ -33,6 +33,11 @@ interface IFrameworkNameProvider
         /// </summary>
         bool TryGetShortProfile(string frameworkIdentifier, string profile, out string profileShortName);
 
+        /// <summary>
+        /// Get the official platform name from the short name.
```

Comment:
Isn't it the other way: "Get the official platform short name from the platform name" ? The out argument is `platformShortName`